meteor/client/ui/SegmentTimeline/SourceLayerItem.tsx

```diff
@@ -102,6 +103,13 @@ export const SourceLayerItem = withTranslation()(
 			return Math.round(this.props.timeScale * time)
 		}
 
+		shouldSwapDurationLabel = (): boolean => {
```

Comment:
I would suggest renaming this to `sourceDurationLabelAlignment`, which returns `left` or `right`; to make the intention of this function more clear. Same goes for the places where this function is used later on.